string topic_in
string file_out
---
string comment
